cypress/integration/parallel-6/savedArticles.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { featureSwitches } from '../../../shared/featureSwitches';
+
+if (!featureSwitches.savedArticles) {
```

Comment:
Think it would be good to have a mechanism which toggles the feature switch state on / off actually in the test rather than reading what's in defined in the code base, so we can check both states at once

Reply:
I've [updated `featureSwitches.ts`](https://github.com/guardian/manage-frontend/pull/996/commits/c152b9607a4cc313b9db7d6259cc8f8448b370a8) to maintain Cypress environment switch states static where required.